src/objects.js

```diff
@@ -1,52 +1,84 @@
 const createPerson = (name, age) => {
-  // your code here
+    const user = {
+        name: (name),
+        age: (age)
+    }
+    return user
```

Comment:
You also didn't necessarily have to assign to a variable here. You could just return an object: `return { ... }`.

One cool thing you can do with objects in ES6, is if your key and variable share the same name, you can omit the colon and the variable.

```js
return {
  name,
  age
}
```

See [Object Literal Property Value Shorthand](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand).